docs/source/notes/amp_examples.rst

```diff
@@ -0,0 +1,158 @@
+.. _amp-examples:
+
+Automatic Mixed Precision Examples
+==================================
+
+.. currentmodule:: torch.cuda.amp
+
+.. contents:: :local:
+
+.. _gradient-scaling-examples:
+
+Gradient Scaling
+^^^^^^^^^^^^^^^^
+
+Gradient scaling helps ensure convergence when training with mixed precision,
+as explained :ref:`here<gradient-scaling>`.
+
+The following code snippets show how :class:`torch.cuda.amp.AmpScaler` can be used to
+perform dynamic gradient scaling automatically.
+
+Working with a Single Optimizer
+-------------------------------
+
+::
+
+    # Create an AmpScaler instance.
+    scaler = AmpScaler()
+    ...
+    for input, target in data:
+        optimizer.zero_grad()
+        output = model(input)
+        loss = loss_fn(output, target)
+
+        # Scale the loss, and call backward() on the scaled loss to create scaled gradients.
+        scaler.scale(loss).backward()
```

Comment:
Greg: this is better than I thought it would be. You're not calling backward on a magical thing, and backward isn't changing somehow. You're calling backwards on a reasonable thing.